private/pkg/transport/http/httpserver/zap.go

```diff
@@ -62,11 +89,13 @@ func newLogEntry(logger *zap.Logger, request *http.Request) *logEntry {
 }
 
 func (l *logEntry) Write(status int, size int, _ http.Header, duration time.Duration, _ interface{}) {
+	httpField := zap.Object(
```

Comment:
Looks like we can simplify this a bit more:

```go
func (l *logEntry) Write(status, size int, _ http.Header, duration time.Duration, _ interface{}) {
	l.logger.Info(
	    "request", 
	     zap.Object(
	        "httpRequest",
		    newHTTPRequestLog(l.request, status, size, duration),
		 ),
		 zap.String("host", l.request.Host),
		 zap.String("path", l.request.RequestURI),
	),
}

```

What do you think @oliversun09?

Reply:
Fixed : )